pages/desktop/user.py

```diff
@@ -117,10 +118,31 @@ def notification_header_text(self):
 
     def click_update_account(self):
         self.selenium.find_element(*self._update_account_locator).click()
-        WebDriverWait(self.selenium, self.timeout).until(lambda s: self.update_message == "Profile Updated")
-
-    def change_hide_email_state(self):
-        self.selenium.find_element(*self._hide_email_checkbox).click()
+        WebDriverWait(self.selenium, self.timeout).until
+        (lambda s: self.update_message == "Profile Updated")
+
+    def hide_email_address(self):
+        email_checkbox = self.selenium.find_element(*self._hide_email_checkbox_locator)
+        WebDriverWait(self.selenium, self.timeout).until
+        (expected_conditions.visibility_of(email_checkbox))
+
+        if email_checkbox.is_selected():
+            raise Exception('Email is already hidden')
+        else:
+            self.selenium.find_element(*self._hide_email_checkbox_locator).click()
+        WebDriverWait(self.selenium, self.timeout).until
+        (lambda s: email_checkbox.is_selected())
+
+    def show_email_address(self):
```

Comment:
This method is not currently being used, so in that case should be included. Having said that, I think it needs to be used (I'll comment more on this later), so we can keep it.